app/src/main/java/ru/netology/fmhandroid/api/PatientApi.kt

```diff
@@ -0,0 +1,56 @@
+package ru.netology.fmhandroid.api
+
+import okhttp3.OkHttpClient
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.*
+import ru.netology.fmhandroid.BuildConfig.BASE_URL
+import ru.netology.fmhandroid.dto.Admission
+import ru.netology.fmhandroid.dto.Note
+import ru.netology.fmhandroid.dto.Patient
+import ru.netology.fmhandroid.enum.PatientStatusEnum
+import java.util.concurrent.TimeUnit
+
+
+interface PatientApi {
+    @GET("patient?patients_status_list=ACTIVE")
+    suspend fun getAllPatients(): Response<List<Patient>>
+
+    @GET("patient")
+    suspend fun getAllPatientsWithAdmissionStatus(
+        @Query("patients_status_list") status: PatientStatusEnum
+    ): Response<List<Patient>>
```

Comment:
А нельзя на что-то подобное заменить?
```suggestion
    @GET("patient")
    suspend fun getAllPatients(
        @Query("patients_status_list") status: PatientStatusEnum = PatientStatusEnum.ACTIVE
    ): Response<List<Patient>>
```
Если здесь нельзя указать значение параметра по умолчанию, то вынеси этот параметр по умолчанию выше - на уровень репозитория